crystal-map-connector/src/main/kotlin/ch/ergon/dope/extension/type/typefunction/IsArray.kt

```diff
@@ -0,0 +1,24 @@
+package ch.ergon.dope.extension.type.typefunction
+
+import ch.ergon.dope.resolvable.expression.unaliased.type.typefunction.isArray
+import ch.ergon.dope.toDopeType
+import com.schwarz.crystalapi.schema.CMField
+import com.schwarz.crystalapi.schema.CMList
+
+@JvmName("isArrayNumber")
+fun isArray(field: CMField<out Number>) = isArray(field.toDopeType())
+
+@JvmName("isArrayString")
+fun isArray(field: CMField<String>) = isArray(field.toDopeType())
```

Comment:
a lot of unused methods in these files

Reply:
Not anymore, I added tests for each of them